js/kucoin.js

```diff
@@ -2385,9 +2389,10 @@ module.exports = class kucoin extends Exchange {
         await this.loadMarkets ();
         const currency = this.currency (code);
         const requestedAmount = this.currencyToPrecision (code, amount);
-        const accountsById = this.safeValue (this.options, 'accountsByType', {});
```

Comment:
@Dan-krm I've fixed it, but next time use something like `inArray` here otherwise, we will crash in Python in runtime.